nbqa/config.py

```diff
@@ -0,0 +1,129 @@
+"""Module responsible for storing and handling nbqa configuration."""
+
+from shlex import split
+from typing import Any, Callable, ClassVar, Dict, List, Optional
+
+from nbqa.cmdline import CLIArgs
+
+
+class ConfigSections:  # pylint: disable=too-few-public-methods
```

Comment:
Is it possible for the `sections` method to not belong to the class and just be a module-level function, so that this can just be a named tuple and we don't have to disable pylint warnings?

Reply:
this is better.